Gemfile

```diff
@@ -2,3 +2,5 @@ source "https://rubygems.org"
 
 gem "rspec"
 gem "httparty"
+gem "pry-byebug"
+gem "activesupport"
```

Comment:
Am I right in thinking that activesupport is included here in order to use the `delegate_missing_to` method? I was [reading up](https://blog.appsignal.com/2019/04/30/ruby-magic-hidden-gems-delegator-forwardable.html) about delegation recently and I learnt about some other options that come with the Ruby standard library:

1. The `Forwardable` module. This requires you to specify the exact methods you want to delegate, unlike `delegate_missing_to`.
2. `SimpleDelegator`, which automatically forwards any missing methods to the object in the constructor. I'm not sure if it would work when you have two objects in the constructor though, as is the case in `Recipe::SyncToLocal` 🤔

Reply:
I might implement a similar thing using Forwardable or SimpleDelegator, to explore the differences between the two.

Reply:
Oh nice, yeah sounds like they'd work too! I'm not too familiar with them.